cloud/azure/actuators/machine/deployment-template.json

```diff
@@ -81,6 +77,10 @@
     "startup_script": {
       "defaultValue": null,
       "type": "string"
+    },
+    "sshKeyData": {
```

Comment:
Can you rename this to `sshPublicKey`? Or is it `ssh_public_key`?

Reply:
Renamed to `sshPublicKey`